app/components/render-zybook/component.js

```diff
@@ -0,0 +1,51 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+    /*
+    @property zybook
+    @type {Object}
+    @default null
+     */
+    zybook: null,
+    classNames: [ 'render-zybook' ],
+
+    /*
+    @event term_description
+    @param {}
```

Comment:
The `@param` for computed properties should describe the properties inside `}.property( ... )`